io.github.kaii_lb.Overskride.json

```diff
@@ -0,0 +1,76 @@
+{
+    "app-id" : "io.github.kaii_lb.Overskride",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "overskride",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--allow=bluetooth",
+        "--socket=system-bus",
+        "--socket=session-bus",
```

Comment:
This isn't allowed by default and I doubt it is correct.

Please try figure out which specific bus names are needed using --log-session-bus, dbus monitor etc.